chat-widget/automation_tests/e2e/pages/livechat.ts

```diff
@@ -0,0 +1,449 @@
+import { BrowserContext, Page } from "playwright";
```

Comment:
filename naming convention should be consistent. liveChat.ts

same for all files.